addon/modifiers/uk-modal.js

```diff
@@ -0,0 +1,57 @@
+import { registerDestructor } from "@ember/destroyable";
+import Modifier from "ember-modifier";
+import UIkit from "uikit";
+
+export default class UkModalModifier extends Modifier {
+  #modal;
+  #events;
+
+  constructor(owner, args) {
+    super(owner, args);
+
+    registerDestructor(this, () => {
+      this.#events.forEach(([event, handler]) => {
+        UIkit.util.off(this.#modal.$el, event, handler);
```

Comment:
What is $e1 in this case?

Reply:
It's `$el` - the element on which the modal was created on